modules/exploits/windows/local/cve_2018_8453_win32k_priv_esc.rb

```diff
@@ -11,24 +11,23 @@ class MetasploitModule < Msf::Exploit::Local
   include Msf::Post::Windows::Priv
   include Msf::Exploit::FileDropper
 
-
   def initialize(info={})
     super(update_info(info,
     'Name'            => 'Windows NtUserSetWindowFNID Win32k User Callback',
     'Description'     => %q{
-      An elevation of privilege vulnerability exists in Windows when the Win32k component
-      fails to properly handle objects in memory, aka "Win32k Elevation of Privilege Vulnerability."
-      This affects Windows 7, Windows Server 2012 R2, Windows RT 8.1, Windows Server 2008, Windows
-      Server 2019, Windows Server 2012, Windows 8.1, Windows Server 2016, Windows Server 2008 R2,
-      Windows 10, Windows 10 Servers.
+        An elevation of privilege vulnerability exists in Windows when the Win32k component
+        fails to properly handle objects in memory, aka "Win32k Elevation of Privilege Vulnerability."
+        This affects Windows 7, Windows Server 2012 R2, Windows RT 8.1, Windows Server 2008, Windows
+        Server 2019, Windows Server 2012, Windows 8.1, Windows Server 2016, Windows Server 2008 R2,
+        Windows 10, Windows 10 Servers.
 
-      This module is tested against Windows 10 v1703 x86.
-    },
+        This module is tested against Windows 10 v1703 x86.
+      },
     'License'         => MSF_LICENSE,
     'Author'          => [
         'ze0r',           # Exploit analysis and PoC
         'Kaspersky Lab',  # Vulnerability discovery/detection
-        'Jacob Robles'    # Metasploit
```

Comment:
I didn't notice that you wrote Metasploit. :joy:

Reply:
Lol. Yeah. Kind of a big error on my part